diff -Naupr a/boreas/CMakeLists.txt b/boreas/CMakeLists.txt
--- a/boreas/CMakeLists.txt	2023-01-09 09:51:09.000000000 +0100
+++ b/boreas/CMakeLists.txt	2023-01-24 14:26:46.665367579 +0100
@@ -57,19 +57,9 @@ endif (NOT PCAP)
 message (STATUS "Looking for pcap-config...")
 find_program (PCAP_CONFIG pcap-config)
 
-if (PCAP_CONFIG)
-  message (STATUS "Looking for pcap-config... ${PCAP_CONFIG}")
-  execute_process (COMMAND pcap-config --libs
-    OUTPUT_VARIABLE PCAP_LDFLAGS
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
-  execute_process (COMMAND pcap-config --cflags
-    OUTPUT_VARIABLE PCAP_CFLAGS
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
-else (PCAP_CONFIG)
   message (STATUS "pcap-config not found, using defaults...")
   set (PCAP_LDFLAGS "-L/usr/lib -lpcap")
   set (PCAP_CFLAGS "-I/usr/include")
-endif (PCAP_CONFIG)
 
 find_package (Threads)
 
